CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
 
+### Removed
+
+* The `app=...` shortcut has been removed. Use the explicit style of `transport=httpx.WSGITransport()` or `transport=httpx.ASGITransport()` instead.
```

Comment:
Since this PR is version 1.0 proposal, why don't you track its changelog in #3069 ?

Reply:
https://github.com/encode/httpx/pull/3069/files#diff-75bb3ae57865eb28e99c3e4fff3efc72a5d49e3f14ac324195b405058cb9731aR13

Reply:
Then do we need to re-mention it in this PR?

Reply:
They should do before approval.